tests/integration/devfile/cmd_devfile_watch_test.go

```diff
@@ -9,13 +9,19 @@ import (
 	. "github.com/onsi/gomega"
 
 	"github.com/openshift/odo/tests/helper"
+	"github.com/openshift/odo/tests/integration/devfile/utils"
 )
 
 var _ = Describe("odo devfile watch command tests", func() {
 	var namespace string
 	var context string
+	var cmpName string
 	var currentWorkingDirectory string
 
+	// TODO: all oc commands in all devfile related test should get replaced by kubectl
+	// TODO: to goal is not to use "oc"
+	oc := helper.NewCliRunner("oc")
```

Comment:
Please run the changes for `kubectl` also as we are running devfile watch on minikube.